pkg/rpcclient/wsclient.go

```diff
@@ -441,12 +441,15 @@ const (
 var ErrNilNotificationReceiver = errors.New("nil notification receiver")
 
 // ErrWSConnLost is a WSClient-specific error that will be returned for any
-// requests after disconnection (including intentional ones via
-// (*WSClient).Close).
+// requests after disconnection (excluding intentional ones made after
+// (*WSClient).Close). For those requests that are done after a call to
+// (*WSClient).Close, see ErrConnClosedByUser.
 var ErrWSConnLost = errors.New("connection lost")
 
-// errConnClosedByUser is a WSClient error used iff the user calls (*WSClient).Close method by himself.
-var errConnClosedByUser = errors.New("connection closed by user")
+// ErrConnClosedByUser is a WSClient error returned iff RPC request is sent after
+// the user calls (*WSClient).Close method by himself. Note that this error won't
+// be returned from (*WSClient).GetError method.
+var ErrConnClosedByUser = errors.New("connection closed by user")
```

Comment:
And this one (shortened to "closed by user") at the same time. It'd be like `connection lost: closed by user`, OK. Can be checked for `ErrWSConnLost` to quickly discover that reconnection is needed, can be checked for `ErrConnClosedByUser` if needed as well.